web/full/library.css

```diff
@@ -15,6 +15,12 @@ body.DesktopUI div[class*="appportraithover_CapsuleBackgroundContainer_"]
 	border-radius: var(--card_radius) !important;
 }
 
+#SteamDesktop .library_Container_3xRRJ div[class*="gamelistentry_Container_"] img[class*="libraryassetimage_Image_"],
+body.DesktopUI div[class*="hoverposition_HoverPosition_"] div[class*="gamelistentry_Container_"] img[class*="libraryassetimage_Image_"]
+{
+	border-radius: var(--button_radius) !important;
```

Comment:
Icon radius should probably be a separate variable.

Reply:
For cleanliness sake sure, though I don't think it'll wind up being used too much.